tools/ts_library_builder/ast_util_test.ts

```diff
@@ -0,0 +1,63 @@
+import { assert, test } from "../../js/testing/testing";
+import {
+  addInterfaceProperty,
+  addSourceComment,
+  addVariableDeclaration,
+  checkDiagnostics,
+  flattenNamespace,
+  getSourceComment,
+  loadDtsFiles,
+  loadFiles,
+  namespaceSourceFile,
+  normalizeSlashes
+} from "./ast_util";
+
+test(function astUtilAddInterfaceProperty() {
+  // TODO build out
+  assert(!!addInterfaceProperty);
+});
+
+test(function astUtilAddSourceComment() {
+  // TODO build out
+  assert(!!addSourceComment);
+});
+
+test(function astUtilAddVariableDeclaration() {
+  // TODO build out
+  assert(!!addVariableDeclaration);
+});
+
+test(function astUtilCheckDiagnostics() {
+  // TODO build out
+  assert(!!checkDiagnostics);
+});
+
+test(function astUtilFlattenNamespace() {
+  // TODO build out
+  assert(!!flattenNamespace);
+});
+
+test(function astUtilGetSourceComment() {
+  // TODO build out
+  assert(!!getSourceComment);
+});
+
+test(function astUtilLoadDtsFiles() {
+  // TODO build out
+  assert(!!loadDtsFiles);
+});
+
+test(function astUtilLoadFiles() {
+  // TODO build out
+  assert(!!loadFiles);
+});
+
+test(function astUtilNamespaceSourceFile() {
+  // TODO build out
+  assert(!!namespaceSourceFile);
+});
+
+test(function astUtilNormalizeSlashes() {
+  // TODO build out
+  assert(!!normalizeSlashes);
+});
```

Comment:
No need to add all these stubs - it's just more to maintain. Just put all the tests into `test.ts`.